bin/lwc-jest.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
```

Comment:
If it's a fil in the `bin` folder you should probably remove the `.js` extension and make sure the file is executable.